python/singa/autograd.py

```diff
@@ -697,15 +833,32 @@ def backward(self, dy):
 
 
 def prelu(x, slope):
+    """
+    PRelu applies the function f(x) = slope * x for x < 0, f(x) = x for x >= 0 to the data tensor elementwise.
+    Args:
+        x: matrix.
+    Return:
+        the result Tensor
+    """
     return PRelu()(x, slope)[0]
 
 
 class Add(Operation):
+    """
+    Performs element-wise binary addition.
+    """
 
     def __init__(self):
         super(Add, self).__init__()
 
     def forward(self, a, b):
+        """
+        Args:
+            a (CTensor): First operand..
```

Comment:
For simple functions, we can simplify the docstring, e.g., no need to use "First operand.." or "Second operand". You can put "Return a+b, where a and b are CTensor."

Reply:
fixed, and for other simple operators, such as or, and, etc, I also change them like this.